app/views/posts/index.html.erb

```diff
@@ -27,8 +34,108 @@
       </tr>
     <% end %>
   </tbody>
-</table>
+  </table>
+</div>
+
+<!--Scroll to top button-->
+
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 20px;
+}
+
+#myBtn {
+  display: none;
+  position: fixed;
+  bottom: 20px;
+  right: 30px;
+  z-index: 99;
+  font-size: 18px;
+  border: none;
+  outline: none;
+  background-color: red;
+  color: white;
+  cursor: pointer;
+  padding: 15px;
+  border-radius: 4px;
+}
+
+#myBtn:hover {
+  background-color: #555;
+}
+</style>
+</head>
+<body>
+
+<button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
+
+<div style="background-color:black;color:white;padding:30px">Scroll Down</div>
+<div style="background-color:lightgrey;padding:30px 30px 2500px">This example demonstrates how to create a "scroll to top" button that becomes visible 
+  <strong>when the user starts to scroll the page</strong>.</div>
+
+<script>
+//Get the button
+var mybutton = document.getElementById("myBtn");
+
+// When the user scrolls down 20px from the top of the document, show the button
+window.onscroll = function() {scrollFunction()};
+
+function scrollFunction() {
+  if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+    mybutton.style.display = "block";
+  } else {
+    mybutton.style.display = "none";
+  }
+}
+
+// When the user clicks on the button, scroll to the top of the document
+function topFunction() {
+  document.body.scrollTop = 0;
+  document.documentElement.scrollTop = 0;
+}
+</script>
+
+</body>
+</html>
+
+ 
+
+
+
+<!--<p id="notice"><%= notice %></p>-->
+
+<!--<h1>Posts</h1>-->
+
+<!--<%= link_to 'New Post', new_post_path %>-->
+
+<!--<table border="1" class="postsTable">-->
+<!--  <thead>-->
+<!--    <tr>-->
+<!--      <th><%= sortPosts "PosterID" %></th>-->
+<!--      <th><%= sortPosts "poster_email" %></th>-->
+<!--      <th><%= sortPosts "Description" %></th>-->
+<!--      <th><%= sortPosts "updated_at" %></th>-->
+<!--      <th colspan="3"></th>-->
+<!--    </tr>-->
+<!--  </thead>-->
 
-<br>
+<!--  <tbody>-->
+<!--    <% @posts.each do |post| %>-->
+<!--      <tr>-->
+<!--        <td><%= post.user_id %></td>-->
+<!--        <td><%= post.poster_email %></td>-->
+<!--        <td><%= link_to post.description, post %></td>-->
+<!--        <td><%= post.updated_at %></td>-->
+<!--        <td><%= link_to 'Edit', edit_post_path(post) %></td>-->
+<!--        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>-->
+<!--      </tr>-->
+<!--    <% end %>-->
+<!--  </tbody>-->
+<!--</table>-->
```

Comment:
For readability, I don't think there is a need to comment out unused code since there is always a way to go back or find previous code with git.

Reply:
Ok thanks for the helpful tip!!